db/migrate/20171107000013_change_view_models.rb

```diff
@@ -0,0 +1,62 @@
+class ChangeViewModels < ActiveRecord::Migration
+  def change
+
+    drop_table :view_attributes
```

Comment:
Can you amend this migration so it does not drop tables? If you are having 'not null' issues, you can set a default and drop it later, e.g.:

alter_table :something do
  add_column :whatever, null: false, default: 'etc'
end
alter_table :something do
  set_column_default :whatever, nil
end